README.md

```diff
@@ -1 +1,23 @@
-# ic-500
\ No newline at end of file
+# Repository for the IC-500: Designing and Managing a ForgeRock<sup>&reg;</sup> Identity Cloud Solution
+
+This repository is for distributing exercise-related files for each lesson of the IC-500 course.
+Students should download these files as needed for the class exercises.
+
+## Chapter 1: Architecting and Designing a ForgeRock<sup>&reg;</sup> Identity Cloud Solution
+
+Below is a brief introduction to each file being used in the chapter one exercises:
+
+- `CH01-ARCHITECT-DESIGN-WORKSHEETS-EXAMPLES.xlsx`
+
+  This file contains worksheets that are populated with sample data.
+  This file is mainly for demonstration purposes.
+  You can download this file if you would like to see sample entries for the different IAM domains.
+
+- `CH01-ARCHITECT-DESIGN-WORKSHEETS-EXERCISES.xlsx`
+
+  This file contains the worksheets for capturing discovery and design data for each of the IAM Domains.
+  Download this worksheet for the exercises and your own personal use.
+  
+- `99-user.ldif`
```

Comment:
Oops, this should be `99-user-pretty.ldif`!

Reply:
I missed that one. Good catch.